src/main/java/com/example/onlinebookstore/model/Status.java

```diff
@@ -0,0 +1,8 @@
+package com.example.onlinebookstore.model;
+
+public enum Status {
```

Comment:
```suggestion
public enum OrderStatus {
```
Or move it to be a nested class of class `Order`